src/java/fr/paris/lutece/plugins/blog/web/adminDashboard/BlogAdminDashboardComponent.java

```diff
@@ -0,0 +1,116 @@
+package fr.paris.lutece.plugins.blog.web.adminDashboard;
+
+import fr.paris.lutece.portal.business.user.AdminUser;
+import fr.paris.lutece.portal.service.dashboard.admin.AdminDashboardComponent;
+import fr.paris.lutece.portal.service.template.AppTemplateService;
+import fr.paris.lutece.util.html.HtmlTemplate;
+import fr.paris.lutece.plugins.blog.business.BlogAdminDashboardHome;
+import fr.paris.lutece.portal.util.mvc.commons.annotations.Action;
+import fr.paris.lutece.portal.util.mvc.admin.annotations.Controller;
+
+import java.util.HashMap;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+
+
+/**
+ * This class provides the user interface to manage general Blog feature in the site admin dashboard ( modify mandatory tag number )
+ */
+@Controller( controllerJsp = "ManageAdminDashboard.jsp", controllerPath = "jsp/admin/plugins/blog/", right = "BLOG_AVANCED_CONFIGURATION" )
+
+public class BlogAdminDashboardComponent extends AdminDashboardComponent
+{
+    private static final long serialVersionUID = 3045411044102177294L;
+    private static final String TEMPLATE_MANAGE_EDITOR_CONFIG = "admin/plugins/blog/dashboard/manage_editor_config.html";
+    private static final String MARK_NUMBER_MANDATORY_TAGS = "number_mandatory_tags";
+    private static final String TEMPLATE_MANDATORY_TAGS = "admin/plugins/blog/dashboard/modify_mandatory_blog_tags.html";
+    private static final String ACTION_UPDATE_MANDATORY_TAG_NUMBER = "updateMandatoryTagNumber";
+    private static final String DASHBOARD_PAGE = "jsp/admin/AdminTechnicalMenu.jsp";
+    private static final String DASHBOARD_PAGE_TAG = "?tab=modifyMandatoryBlogTagNumber";
+    private static final String ACCESS_DENIED_MESSAGE = "portal.message.user.accessDenied";
+    public static final String RIGHT_AVANCED_CONFIGURATION = "BLOG_AVANCED_CONFIGURATION";
+    private static final String PARAMETER_NUMBER_MANDATORY_TAGS = "numberMandatoryTag";
+    private static final String TEMPLATE_MANAGE_MAX_PUBLICATION_DATE = "admin/plugins/blog/dashboard/manage_maximum_publication_date.html";
+    private static final String PARAMETER_MAX_PUBLICATION_DATE_VALUE = "maxPublicationDate";
+    private static final String ACTION_MANAGE_MAX_PUBLICATION_DATE = "manageMaxPublicationDate";
+    private static final String DASHBOARD_PAGE_MAX_PUBLICATION_DATE = "?tab=manageMaxPublicationDate";
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String getDashboardData( AdminUser user, HttpServletRequest request )
```

Comment:
Can you handle all three cases in a single template and avoid concatenation to make the management of template elements more flexible.
We will also gain in terms of performance by calling the template engine only once, which is much more efficient